src/main/java/com/google/sps/app/servlets/NewTripServlet.java

```diff
@@ -120,17 +121,17 @@ public Trip getTrip(HttpServletRequest request){
         Date end = DataStoreHelper.parseInputDate(request.getParameter(END_DATE_PARAM));
         String participantInput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
         List<String> participantIds = new ArrayList<String>();
-        List<String> checkerduplicate = new ArrayList<String>();
+        List<String> names = new ArrayList<String>();
         List<String> participants = DataStoreHelper.splitUserList(participantInput);
-        checkerduplicate.add(request.getParameter(USERNAME_PARAM));
+        names.add(request.getParameter(USERNAME_PARAM));
         participantIds.add(request.getParameter(USERID_PARAM));
         for (String name: participants){
```

Comment:
Can you fix the indentation for this for loop. seems like there is a few extra tabs